templates/base/head_script.tmpl

```diff
@@ -46,3 +46,4 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 	{{/* in case some pages don't render the pageData, we make sure it is an object to prevent null access */}}
 	window.config.pageData = window.config.pageData || {};
 </script>
+<script src="{{AssetUrlPrefix}}/js/custcomp.js?v={{AssetVersion}}" onerror="alert('Failed to load asset files from ' + this.src + '. Please make sure the asset files can be accessed.')"></script>
```

Comment:
Isn't there already a similar `alert` somewhere else?

Reply:
Yes, but I don't think it is either possible nor a good idea to depend on another function inside the `failed to load` handler…

Reply:
Yes, but I don't think it is either possible nor a good idea to depend on another function inside the `failed to load` handler…

Reply:
I'm more worried about the possibilty of multiple chained alerts, not code re-use.

Reply:
IIRC, the other error was recently moved to the admin page, so users won't even see it anymore.

Reply:
> Isn't there already a similar `alert` somewhere else?

Yes, for the  `index.js`


> I'm only worried about the possibilty of multiple chained alert popups, not code re-use.

Only if the user's site has  been broken seriously, nothing works.

----

I can  only put the  error message on the `index.js` if you like.

Reply:
Maybe collect failed assets in an array, and display one message for all?

Reply:
I do not feel it's necessary to make it so complex.